pages/nft-book-store/new.vue

```diff
@@ -14,15 +14,30 @@
       <p>Total number of NFT for sale: {{ totalStock }}</p>
       <hr>
       <div v-for="p, index in prices" :key="index">
-        <p><label>Price(USD) per NFT Book</label></p>
-        <input :value="p.price" type="number" @input="e => updatePrice(e, 'price', index)">
+        <p><label>Price(USD) per NFT Book (Minimal $5)</label></p>
+        <input :value="p.price" type="number" min="5" @input="e => updatePrice(e, 'price', index)">
```

Comment:
Should use `MINIMAL_PRICE`?
```suggestion
        <input :value="p.price" type="number" min="MINIMAL_PRICE" @input="e => updatePrice(e, 'price', index)">
```